src/MOI/MOI_callbacks.jl

```diff
@@ -7,17 +7,20 @@ Set a generic Xpress callback function.
 struct CallbackFunction <: MOI.AbstractCallback end
 
 function MOI.set(model::Optimizer, ::CallbackFunction, ::Nothing)
+
     if model.callback_data !== nothing
-        removecboptnode(model.inner, C_NULL, C_NULL)
+        Lib.XPRSremovecboptnode(model.inner, C_NULL, C_NULL)
         model.callback_data = nothing
     end
     model.has_generic_callback = false
     return
 end
 
 function MOI.set(model::Optimizer, ::CallbackFunction, f::Function)
+
     if model.callback_data !== nothing
-        Xpress.removecboptnode(model.inner, C_NULL, C_NULL)
+        Lib.XPRSremovecboptnode(model.inner, C_NULL, C_NULL)
+        
         model.callback_data = nothing
     end
```

Comment:
@RaphMota in the changes there are a lot of new lines added that do not follow the pattern that was there previously.

Reply:
I just removed the new lines, but there are some functions as getobj or getqrowqmatrixtriplets for which I had to add some lines in order to make them work. Do i have to remove them too?

Reply:
If the lines are necessary they have to stay.
@guilhermebodin is talking about the new blank lines that are not necessary.